### ISSUE TITLE
Question: Keypoints for Humans

### ISSUE THREAD

**Issue by tgorm1965** [ZumoLabs/zpy#133]
Hey this is more of a question. 

I wanted to understand how to use the keypoint annotation python file within ZPY to create keypoint annotations. Specifically, these annotations: head pose (6DOF), facial landmarks (3D, 2D), gaze direction (vector), body pose / human joints

It looks like there are some components there but it'd be helpful to get your take.

It would also be amazing if you could show some code examples! I look forward to your reply.

**Comment by HugoCMU** [comment]
There are many different ways of doing keypoint annotations for humans, which also depend on what humans you are using. For our projects we have used [Mixamo](https://www.mixamo.com/#/?page=1&type=Character) and [Anima](https://secure.axyz-design.com/en/shop/category/free-3d-people) as sources for synthetic humans. I would recommend Mixamo as it is easy to work with and there is a good library of animations to go along with the rigged humans.

Keypoints are usually reported as XYV (xy position of a keypoint in image pixel space, and visibility V) or XYZ (xyz position of a keypoint in world space). Connectivity refers to how each of the keypoints are connected to each other, and changes depending on the annotation format e.g. COCO or Body25B.

I uploaded an example sim run script that uses the keypoint abstraction: https://github.com/ZumoLabs/zpy/commit/fd1d2814306a075feeba209cd24581818d8d846a
